MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1121,7 +1121,7 @@ public UUID getId() {
     }
 
     public boolean isChild(final LocalDate today) {
-        return getAge(today) <= 13;
+        return getAge(today) <= 18;
```

Comment:
I guess that should be < 18, right?

Reply:
Yeah, I don't really know what happened here, because it's `return getAge(today) <= 18` locally. Either way, easy fix.